README.md

```diff
@@ -9,10 +9,61 @@ WSO2 Balana Implementation
 
 ---
 
-This is WSO2 Balana, The Open source XACML implemantation which supports XACML 3.0,2.0,1.1 and 1.0 versions. WSO2 Balana is based on Sun's XACML Implementation [1]. License of Sun's XACML implementation can be found at here [2]. But WSO2 Balana implementation is released under Apache2 license [3]
+## Welcome to WSO2 Balana
+Balana is WSO2's open source implementation of the XACML specification building on Sun's XACML Implementation[1]. As the name suggests Balana(the fortress) is a powerful entitlement engine to externalize authorization from your applications. With it's modular architecture you can easily develop a fully fledged entitlement solution in no time.
```

Comment:
Let's add hyperlinks to XACML specifications

Reply:
Yes, let's do so.